chapter3/sample_app/test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+class StaticPagesControllerTest < ActionDispatch::IntegrationTest
+  test "should get home" do
```

Comment:
Here at Docfox we follow specific format when writing test descriptions.
Please change your descriptions to the below format
```suggestion
  test "GET #home should render the root page, a success response, and a correct title " do
```